qemu/tests/uefi_pkg.py

```diff
@@ -0,0 +1,70 @@
+import re
+import json
+import logging
+
+from avocado.utils import process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Verify descriptor meta-files after ovmf package installation on the host:
+    1) Check edk2-ovmf package has been installed already.
+    2) Qurey file list on edk2-ovmf package.
+    3) Make sure the descriptor meta-files will be in file list.
+       2 files in total.
+       For rhel7: 50-ovmf-sb.json, 60-ovmf.json
+       For rhel8: 40-edk2-ovmf-sb.json, 50-edk2-ovmf.json
+    4) Check the JSON files internally.
+       check that the "filename" elements in both files point to valid files.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def check_element(filename, file_list):
+        """
+        check 'filename' element point to a valid file
+
+        :param filename: 'filename' element
+        :param file_list: query files command output
+        """
+        if filename not in file_list:
+            test.fail("The 'filename' element in meta-file point to an "
+                      "invalid file. The invalid file is '%s'" % filename)
+
+    query_package = params["query_package"]
```

Comment:
from virttest import utils_package

maybe you can found required API in this module.

Reply:
The package name is uncertain. It's name contains timestamp.
I check the module utils_package.
The core functions are package_install, package_remove and operate. And the return value of function \"operate\" is True or False.
I must get output from \"rpm -ql packagename \".
These requirements can not be handled by module utils_package.
So I decide not to use it. Many thanks.